### DISCUSSION TITLE
~/.config/ytfzf/scrapelist   how do I use it?

### DISCUSSION THREAD

**Discussion by plum** [pystardust/ytfzf#731] [General]
I created the 'scrapelist'  with the follwing content in config directory, but cant find how to invoke it. I assume its supposed to scrape (search) for frequent  or habitual searches.  If you can point the way , I'd be grateful. Or just correct my misunderstanding!

```
# youtube this is a search 

odysee theduran 

odysee thenewatlas

odysee southfront
```

CHeers, John

**Comment by Euro20179** [comment]
I don't remember why I added this,
The manual isn't very helpful
The code for scrapelists isn't even in the program lol,

Knowing me, it probably just scraped everything in scrapelist all at once.

if that's true there is multi-scrape, (it doesn't read from a file) but performs a similar task

an example of multi-scrape `ytfzf -cM :NEXT -cO odysee search 1 :NEXT -cO odysee search 2 :NEXT youtube search`

each item seperated by :NEXT essentially acts like a new call to ytfzf (so hopefully most scraping options work)

**Reply by Euro20179** [reply]
also I should mention, if you have `enable_hist=1` in your ytfzf config, it will save your searches to `~/.cache/ytfzf/search_hist` and you can use `ytfzf -q` to use a search query from the `search_hist` file

**Comment by plum** [comment]
@Euro20179  > I don't remember why I added this, The manual isn't very helpful The code for scrapelists isn't even in the program lol,
> 
Ah, I see. Lucky I contacted you!
> 
> if that's true there is multi-scrape, (it doesn't read from a file) but performs a similar task
> an example of multi-scrape `ytfzf -cM :NEXT -cO odysee search 1 :NEXT -cO odysee search 2 :NEXT youtube search`
> each item seperated by :NEXT essentially acts like a new call to ytfzf (so hopefully most scraping options work)

THat'll do instead.
Thanks for a great programme.
John
